app/src/main/java/ch/blinkenlights/android/vanilla/JumpToTimeDialog.java

```diff
@@ -101,6 +103,20 @@ public void afterTextChanged(Editable s) {
 			.create();
 		hoursView.requestFocus();
 		dialog.getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_VISIBLE);
+
+		dialog.setOnShowListener(new DialogInterface.OnShowListener(){
+			@Override
+			public void onShow(DialogInterface dialog) {
+				AlertDialog d = (AlertDialog) dialog;
+				Button bN = d.getButton(DialogInterface.BUTTON_NEGATIVE);
+				Button bP = d.getButton(DialogInterface.BUTTON_POSITIVE);
+
+				ThemeHelper.setAccentColor(bN,bP);
+				ThemeHelper.setAccentColor(hoursView,minutesView,secondsView);
```

Comment:
So we need to manually set the color for every dialog and ui element? That seems cumbersome..

Reply:
Only for specific elements. The dialog is quiet a hassle because i cannot easily access its elements ( i have to wait for "show" to be called, therefore the callback) The only way to set the colors on the buttons i have found so far is to get it from the dialog directly, and therefore i need to style them manually. It's really a shame that we cant influence themes from the runtime, that would make this so much easier

Reply:
That approach will turn in an endless amount of hacks if we have to intercept random UI elements.

Reply:
What are you suggesting instead? We cant influence themes, so we need to set it manually. Either that, or no custom colors at all